pkg/apis/contrail/v1alpha1/tests/config_test.go

```diff
@@ -1,25 +1,16 @@
 package contrailtest
```

Comment:
I think you can also remove the device manager config string from this file, as it is not used anymore. Line 529
https://github.com/Juniper/contrail-operator/blob/943581329a30321da02f2df98cd3489bb009c8d3/pkg/apis/contrail/v1alpha1/tests/config_test.go#L529